core/src/main/java/com/netflix/conductor/validations/WorkflowTaskTypeConstraint.java

```diff
@@ -271,19 +270,6 @@ private boolean isWaitTaskValid(
                 context.buildConstraintViolationWithTemplate(message).addConstraintViolation();
                 valid = false;
             }
-
-            try {
```

Comment:
Don't we still need this check if the values are not parameters? Instead of removing the check, can this be changed to check if the value of `duration` or `until` is a parameter and if its not, then perform these checks?